print/cups/Portfile

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        OpenPrinting cups 2.4.2 v
+github.tarball_from releases
+distname            ${github.project}-${github.version}-source
+revision            0
+categories          print
+license             Apache-2
+maintainers         nomaintainer
+description         CUPS is the current standards-based, open source printing system developed by \
+                    OpenPrinting for Unix-like operating systems and uses IPP Everywhere™ to support \
+                    printing to local and network printers.
+long_description    OpenPrinting CUPS is the most current version of CUPS, a standards-based, \
+                    open source printing system for Unix-like operating systems. CUPS supports \
+                    printing to AirPrint™ and IPP Everywhere™ printers, \
+                    Network and local (USB) printers with Printer Applications, and \
+                    Network and local (USB) printers with (legacy) PPD-based printer drivers. \
+                    CUPS provides the System V ("lp") and Berkeley ("lpr") command-line interfaces, \
+                    a configurable web interface, a C API, and common print filters, drivers, and backends \
+                    for printing. The cups-filters project provides additional filters and drivers.
+homepage            https://openprinting.github.io/cups/
+
+checksums           rmd160  58a24fe177cfb66157899b6a53a90972a6080e74 \
+                    sha256  f03ccb40b087d1e30940a40e0141dcbba263f39974c20eb9f2521066c9c6c908 \
+                    size    8128785
+
+depends_build       port:gmake \
+                    port:autoconf \
+                    port:pkgconfig
+depends_lib         port:zlib
+
+patch {
+    reinplace "s|/System/Library/LaunchDaemons||g" configure
+}
+
+configure.cflags-append     -I${prefix}/include
+configure.cppflags-append   -I${prefix}/include
+configure.cxxflags-append   -I${prefix}/include
```

Comment:
Is `-I${prefix}/include` really needed in CFLAGS and CXXFLAGS? Normally it's only needed in CPPFLAGS.

You don't need to add it to CPPFLAGS yourself; MacPorts does it automatically.